src/pages/en/science/index.html

```diff
@@ -1,6 +1,6 @@
 ---
 lang_de: false
-page-title: Open-Source Project Corona-Warn-App - Blog
+page-title: Open-Source Project Corona-Warn-App - Science Blog
 page-description: Blog for the Corona-Warn-App. Here you get current information about the development of the Corona-Warn-App and all important updates.
```

Comment:
The page-description is also incorrect. Will you change this as well?

Reply:
I‘ll change this as soon as I have the time. Thanks for pointing that out.